pytorch/inference/docker/2.2/py3/Dockerfile.graviton.cpu

```diff
@@ -0,0 +1,253 @@
+    libxext6 \
+    libxrender-dev \
+    openjdk-17-jdk \
+    openssl \
+    scons \
+    vim \
+    wget \
+    unzip \
+    zlib1g-dev \
+    libjpeg-dev \
+    libpng-dev \
+ && apt-get autoremove -y \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+ # Set default gcc, python and pip versions
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 1 && \
+    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 1 && \
+    update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-10 1
+
+# Install OpenMPI
+RUN wget https://www.open-mpi.org/software/ompi/v4.1/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+ && cd openmpi-$OPEN_MPI_VERSION \
+ && ./configure --prefix=/home/.openmpi \
+ && make all install \
+ && cd .. \
+ && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# The ENV variables declared below are changed in the previous section
+# Grouping these ENV variables in the first section causes
+# ompi_info to fail. This is only observed in CPU containers
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
+
+# Install CondaForge miniconda
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-aarch64.sh \
+ && chmod +x ~/mambaforge.sh \
+ && ~/mambaforge.sh -b -p /opt/conda \
+ && rm ~/mambaforge.sh \
+ && /opt/conda/bin/conda install -c conda-forge -y python=$PYTHON_VERSION \
+    ruamel_yaml \
+    cython \
+    ipython \
+    parso \
+    # Below 2 are included in miniconda base, but not mamba so need to install
+    conda-content-trust \
+    charset-normalizer \
+    requests \
+ && conda clean -ya
+
+# Install Common python packages
+RUN ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && pip install \
+    boto3 \
+    scipy \
+    opencv-python \
+    "pyOpenSSL>=24.0.0" \
+    "cryptography>=41.0.2" \
+    "ipython>=8.10.0,<9.0" \
+    "awscli<2" \
+    "urllib3>=1.26.18,<2"
+
+# Ensure PyTorch did not get installed from Conda or pip, prior to now
+RUN pip uninstall -y torch torchvision torchaudio torchdata model-archiver multi-model-server
+
+# install pytorch wheels
+RUN pip install --no-cache-dir -U \
+    ${TORCH_URL} \
+    ${TORCHVISION_URL} \
+    ${TORCHAUDIO_URL} \
+    ${TORCHTEXT_URL}
+
+# Install TorchServe pypi dependencies directly from their requirements.txt file
+# NOTE: This also brings in unnecessary cpu dependencies like nvgpu
+# -> Numpy is higer version as required by torch
+RUN pip install --no-cache-dir -U -r https://raw.githubusercontent.com/pytorch/serve/v${TORCHSERVE_VERSION}/requirements/common.txt \
+ && pip install --no-cache-dir -U \
+    "numpy>=1.22.2,<1.23" \
```

Comment:
Any reason why we're pinning this? For PT2.2 x86 release, the pin is no longer needed and was move to the `conda install` step [here](https://github.com/aws/deep-learning-containers/blob/f237519a5e6f1dcf9204610bab167f8417e235ca/pytorch/inference/docker/2.2/py3/cu118/Dockerfile.gpu#L186). This might also be the case for Graviton as well so I'd consider doing the same.